### ISSUE TITLE
Add Support for E-mail Notification

### ISSUE THREAD

**Issue by SwayerPT** [hdnpt/geartrack-website#70]
Hey Guys,

You should implement support for receiving notifications by e-mail and SMS.
Every new status, every new update.

**Comment by carlosflorencio** [comment]
Hi @SwayerPT, yeah its a much requested feature. 

I'll try to work on it soon. Sending notifications (email, push notifications, etc) will most probably incur in some costs for us (we'll need to pay a provider to send notifications).

Is this a feature you would pay for? It should be a very small amount (one time or yearly).

I'm thinking about the following business plan:
- Notifications for 1 order are free
- > 1 order and you should pay something (to cover our costs)

Another option instead of charging $ (because I know, no one likes to pay for services) is to increase the number of ads displayed for that user, and for those who use notifications, we don't allow the use of an adblock extension. Also, the notifications would not include the new tracking information but a link to see the updated tracking on the app.

What's your opinion?

**Comment by SwayerPT** [comment]
My opinion is... Making something to pay is hard this days... as u know.
But... you should add a subscription feature that gives access for something like....

**Free Account:**

- Add up to 10 orders. 
(Should have a FIELD on DB like "Nº of orders Available to add");

- Important Notification (E-mails): 
For 3 orders (limited);
Add a new Box under the list box only for IMPORTANTS.
(You should MARK in the listbox the one. Then the system E-mail Notify will notify in every 2h(IF HAVE NEW REQUESTS) and will be sent ONLY for IMPORTANTS (this will save delays/requests on your servers by EMAILS)

- Option to control every order (instead of showing all carriers, only show the MOST UPDATED, or EXPAND or decrease the ORANGE box definitly...)

- Create a PROFILE for every user.

- A Mini Forum (something easy, just TOPICS to take advantage and grow up the community.

**Subscribed Account (Could be purchased with Coins):**

- Add up to 20 orders. 
(Should have a field like "Nº of orders Available to add)

- Important Notification (E-mails): 
For 6 orders ONLY;

- Credit System (just to give a chance to Win some coins to buy a New subscription, instead using CASH... (yeahh nobody wants to pay... everybody wins...)
Features for the profile...

- User Coin Gain: (Win Coins helping others when others MARK as HELPFULL ONLY).

- Invite System: inviting +1 friends will add +1 orders to the FIELD on DB "Nº of orders Available to add" for every friend.

- API System: This will bring features for every people who wants to Add your System: ADD, DELETE, UPDATE to their websites or apps... (helping u with more users by tracking).

*You dont need a provider, you just need a LOOPING for every order with restrictions... 
Every Hosting have a PHMAILER on it. Or use https://github.com/PHPMailer/PHPMailer

**Comment by carlosflorencio** [comment]
Yeah this is a hard subject, but we'll launch the notifications feature without a paywall at first. Later down the line we should include some monetization scheme to pay for the costs, your ideas will come in handy 💪.
